svghmi/widget_image.ysl2

```diff
@@ -0,0 +1,48 @@
+// widget_image.ysl2
+
+widget_desc("Image") {
+    longdesc
+    ||
+    If Image widget is a svg:image element, then xlink:href content is replaced by
+    value of given variable.
+    ||
+
+    shortdesc > Image display
+}
+
+widget_class("Image")
+    ||
+        frequency = 5;
+        dispatch(value, oldval, index) {
+            this.fields[index] = value;
```

Comment:
"fields" array is not needed here. Store only if index == 0.
```suggestion
            if(index==0)
                this.given_url = value;
```
Idem for "readyfields", only "ready" member should be needed